tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -3582,3 +3586,535 @@ def test_15_barbican(self):
             reporting.set_test_script_status(tvaultconf.FAIL)
         finally:
             reporting.test_case_to_write()
+
+    # Test case automated #OS-2025
+    # http://192.168.15.51/testlink/linkto.php?tprojectPrefix=OS&item=testcase&id=OS-2025
+    @decorators.attr(type='workloadmgr_api')
+    def test_16_barbican(self):
+        try:
+            test_var = "tempest.api.workloadmgr.barbican.with_secret_order_"
+            tests = [[test_var+"create_secret", 0],
```

Comment:
These are steps in test case but here those are reported as different  test cases

Reply:
We are executing scenario consisting of below tests. Could not attach html report as it is throwing error, hence attached snippet of test execution.

1. tempest.api.workloadmgr.barbican.with_secret_order_create_secret 
2. tempest.api.workloadmgr.barbican.with_secret_order_workload_api
3. tempest.api.workloadmgr.barbican.with_secret_order_full_snapshot_api 
4. tempest.api.workloadmgr.barbican.with_secret_order_incremental_snapshot_api 
5. tempest.api.workloadmgr.barbican.with_secret_order_snapshot_mount_api 
6. tempest.api.workloadmgr.barbican.with_secret_order_filesearch_api
7. tempest.api.workloadmgr.barbican.with_secret_order_selectiverestore_api
8. tempest.api.workloadmgr.barbican.with_secret_order_inplacerestore_api 

tempest.api.workloadmgr.barbican.with_secret_order_oneclickrestore_api